src/site/markdown/server/manage-webadmin.md

```diff
@@ -3269,16 +3269,17 @@ One can await the end of a task, then receive it's final execution report.
 That feature is especially usefull for testing purpose but still can serve real-life scenari.
 
 ```
-curl -XGET http://ip:port/tasks/3294a976-ce63-491e-bd52-1b6f465ed7a2/await
+curl -XGET http://ip:port/tasks/3294a976-ce63-491e-bd52-1b6f465ed7a2/await?timeout=numberOfSeconds
```

Comment:
as you divide by 1000 it's millis, no?
BTW what about allowing some timeunit here? I guess we already have all the needed available code for that.

Reply:
I've found nothing regarding that.

Anyway, I think it clearly not worth the time and effort maintaining such code and a decent suite of tests.

Reply:
Is that new timeout parameter now compulsory?

Reply:
org.apache.james.util.DurationParser?

Reply:
> Is that new timeout parameter now compulsory?

no

Reply:
> org.apache.james.util.DurationParser?

spaces are mandatory, it look likes a bad idea

Reply:
that's not how I read it. But maybe some missing tests